README.md

```diff
@@ -2,27 +2,35 @@
 
 Web app created for comparing product price and offers.
 
-# Installation
-First you need to install MongoDB Community Server from official site https://www.mongodb.com/try/download/community
-Make sure to install also MongoDB Compass to graphical see and manage your data, but it's not required
-After successfull installation add Monogo to PATH variable by:
+## Project setup
+
+### Mongo (Windows)
+
+1. Install MongoDB Community Server from the [official site](https://www.mongodb.com/try/download/community)
+2. Add Monogo to `PATH`:
+
 - go to enviornment variables
 - change environmanet variables
 - in system variables look for PATH
-- add new entry your_path_to_mongo\MongoDB\Server\your_version
+- add new entry `your_path_to_mongo\MongoDB\Server\your_version`
+
+3. Create folder `data/db` in main C folder (your path should look like that `C:\data\db`)
+4. Run CMD in this directory `mongod` command
+
+### Django and virtual env
 
-After that steps create folder data/db in main C folder (your path should look like that C:\data\db)
-Run CMD in this directory `mongod` command
+1. Make sure you're in the parent `kernel` directory
+2. Create virtual environment `python3 -m venv .venv`
+3. Install dependencies `pip install requirements.txt`
+4. Run migrations `python manage.py migrate`
```

Comment:
This is the crucial missing instruction, the rest are just minor optimizations for readability